src/components/input-field/partial-styles/outlined-style-overrides.scss

```diff
@@ -0,0 +1,63 @@
+.mdc-text-field {
+    transition: background-color 0.2s ease;
+
+    &:not(.mdc-text-field--disabled) {
+        background-color: $background-color-normal;
+
+        &:hover {
+            background-color: $background-color-hovered;
+        }
+    }
+
+    &.mdc-text-field--focused {
+        background-color: $background-color-focused;
+    }
+
+    &.mdc-text-field--disabled {
+        background-color: $background-color-disabled;
+    }
+
+    &:not(.mdc-text-field--disabled) {
+        .mdc-notched-outline__leading,
+        .mdc-notched-outline__notch,
+        .mdc-notched-outline__trailing {
+            transition: border-color 0.2s ease;
```

Comment:
It sounds really nice. I don't have that much "koll på" these available options. Also I'd like to take it in a separate issue. Since we have many of these custom animations generally, and I have to educate myself first.

Reply:
Yeah, certainly a separate issue 👍